Gemfile

```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 
-gem 'rails', '~> 4.0'
+gem 'rails', '~> 5.0.0'
+gem 'rails-controller-testing'
```

Comment:
I think we should keep the dependency. Since we're upgrading to Rails 5, it will make tests passing on both Rails 4 and 5, otherwise we'd have to manually update all controller tests for Rails 5. The gem itself is not too heavy.